coloring.py

```diff
@@ -75,17 +95,32 @@ def colors(self) -> List[int]:
 
         :return: a list of colors of the coloring
         """
-        return self._dict.keys()
+        return list(self._dict.keys())
 
     @property
-    def num_colors(self) -> int:
+    def vertices(self) -> List[Vertex]:
+        """
+        Returns the vertices in the coloring
+
+        :return: list of vertices in the coloring
+        """
+        return list(self._vertex_dict.keys())
+
+    def __len__(self) -> int:
         """
         Returns the number of color classes in the coloring
 
         :return: the number of color classes in the coloring
         """
         return len(self._dict.keys())
 
+    def items(self):
+        """
+        Return a copy of a set of the (color,List[Vertex]) pairs in the Coloring
+        :return: set of (color,DoubleLinkedList()) pairs
```

Comment:
Hoe zorgt dit ervoor dat het een (color, DoubleLinkedList())-pair is?

Reply:
Helaas alleen impliciet (doordat het zo gemaakt wordt in de class), maar ik kon niet vinden wat het type output is van de standaard dict.items().